src/ngraph/frontend/onnx_import/utils/common.hpp

```diff
@@ -68,53 +68,6 @@ namespace ngraph
                 return range;
             }
 
-            /// \brief      Handle out of range axis.
-            ///
-            /// \param[in]  node         The node with requested axis.
-            /// \param[in]  axis         The requested axis value.
-            /// \param[in]  tensor_rank  The corresponding tensor rank.
-            ///
-            /// \return    Checking if axis is in range [-tensor_rank, tensor_rank-1], otherwise
-            /// returns error.
-            ///            If negative axis, it counts from the last to the first axis, by adding
-            ///            tensor_rank to axis.
-            ///
-            std::size_t validate_axis(const ngraph::onnx_import::Node& node,
```

Comment:
This function should be kept. You can rename it to `normalize_axis` if you need. This function in let us clearly deal with ONNX Importer objects.

Reply:
I think that handling one version of `normalize_axes` in `onnx_importer` is more confusing. What's more this version can be useful too for nGraph